ui/src/components/admin/users/AdminUser.js

```diff
@@ -260,13 +317,20 @@ export default function AdminUser(props) {
       headerName: 'Personal Details',
       marryChildren: true,
       children: [
-        { headerName: 'Phone', field: 'phoneNumber' },
+        {
+          headerName: 'Phone',
+          field: 'phoneNumber',
+          valueSetter: phoneNumberSetter,
+          // Didn't enable NUMBER filter because there is no "contains" option for number filters: not as user-friendly as TEXT filter
```

Comment:
As per the in-line comment:
Text filtering for phone numbers is more powerful and perhaps more relevant. The admin doesn't have to know the exact phone number and can use the "contains" filter option. (number type filter options don't include "contains")